backend/src/telegram/routes/telegram-campaign.routes.ts

```diff
@@ -76,7 +76,9 @@ const tesseractCampaignValidator = {
     url: Joi.string()
       .trim()
       .custom((value: string, helpers: any) => {
-        const url = value.match(/^https:\/\/storage(-test)?.vault.gov.sg\/.+$/g)
+        const url = value.match(
+          /^https?:\/\/storage(-test)?.vault.gov.sg\/.+$/g
```

Comment:
Shall we make this a variable in `config` with current value as the default value? In case, there's a change we wouldn't need to update the code.